### PR TITLE
Refactor & Fix pr_cmds.cpp

### PR DESCRIPTION
**Pull request by In-line**

### CODE REVIEW

**Comment by LevShisterov** [comment]
Please move 8848ef3 into another PR if you wish. I am not ready to check it in.

**Comment by In-line** [comment]
@LevShisterov One second.

**Comment by LevShisterov** [comment]
Thanks!

File: rehlds/engine/pr_cmds.cpp [hunk 1]
```diff
@@ -828,7 +828,7 @@ void EXT_FUNC PF_stuffcmd_I(edict_t *pEdict, const char *szFmt, ...)
 	Q_vsnprintf(szOut, sizeof(szOut), szFmt, argptr);
 	va_end(argptr);
 
-	szOut[1023] = 0;
+	szOut[sizeof(szOut) - 1] = 0;
```
Comment (WPMGPRoSToTeMa) [inline rehlds/engine/pr_cmds.cpp]: Ideally should be `ARRAYSIZE`.

**Pull request merged**